src/ray/gcs/gcs_actor_scheduler.cc

```diff
@@ -521,27 +531,28 @@ void GcsActorScheduler::DoRetryCreatingActorOnWorker(
   }
 }
 
-std::shared_ptr<RayletClientInterface> GcsActorScheduler::GetOrConnectRayletClient(
-    const rpc::Address &raylet_address) {
-  return raylet_client_pool_.GetOrConnectByAddress(raylet_address);
-}
-
-bool GcsActorScheduler::KillActorOnWorker(const rpc::Address &worker_address,
-                                          ActorID actor_id) {
-  if (worker_address.node_id().empty()) {
-    RAY_LOG(DEBUG) << "Invalid worker address, skip the killing of actor " << actor_id;
+bool GcsActorScheduler::CleanupWorkerForActor(const rpc::Address &raylet_address,
+                                              const rpc::Address &worker_address,
+                                              ActorID actor_id) {
```

Comment:
please add an `INFO` level log that an actor is being killed (unless it's already logged in the callsite)